app/src/test/java/com/x8bit/bitwarden/data/autofill/fido2/manager/Fido2CredentialManagerTest.kt

```diff
@@ -536,6 +538,53 @@ class Fido2CredentialManagerTest {
             )
         }
 
+    @Suppress("MaxLineLength")
+    @Test
+    fun `registerFido2Credential should return Error when getSignatureFingerprintAsHexString is null`() =
+        runTest {
+            val mockSigningInfo = mockk<SigningInfo> {
+                every { hasMultipleSigners() } returns true
+            }
+            val mockFido2CredentialRequest = createMockFido2CredentialRequest(
+                number = 1,
+                signingInfo = mockSigningInfo,
+            )
+
+            val result = fido2CredentialManager.registerFido2Credential(
+                userId = "mockUserId",
+                fido2CredentialRequest = mockFido2CredentialRequest,
+                selectedCipherView = createMockCipherView(number = 1),
+            )
+
+            assertTrue(
+                result is Fido2RegisterCredentialResult.Error,
```

Comment:
Can we use `assertEquals` here and actually verify the contents

Reply:
changed it also on a couple other places that are similar

Reply:
Awesome!